ext/pcntl/pcntl.c

```diff
@@ -385,6 +401,42 @@ PHP_FUNCTION(pcntl_waitpid)
 }
 /* }}} */
 
+/* {{{ Waits on or returns the status of a forked child as defined by the waitid() system call */
```

Comment:
nit: no need to bother with these with new functions, you could not know but we no longer need these.